app/src/main/java/org/mozilla/fenix/components/metrics/LeanplumMetricsService.kt

```diff
@@ -83,12 +84,19 @@ class LeanplumMetricsService(private val application: Application) : MetricsServ
         leanplumJob = scope.launch {
 
             val applicationSetLocale = LocaleManager.getCurrentLocale(application)
-            val currentLocale = when (applicationSetLocale != null) {
-                true -> applicationSetLocale.isO3Language
-                false -> Locale.getDefault().isO3Language
-            }
-            if (!isLeanplumEnabled(currentLocale)) {
-                Log.i(LOGTAG, "Leanplum is not available for this locale: $currentLocale")
+            val currentLocale = applicationSetLocale ?: Locale.getDefault()
+            val languageCode =
+                currentLocale.iso3LanguageOrNull
+                ?: currentLocale.language.let {
```

Comment:
I double-checked this as `language` is annotated with `@RecentlyNotNull` and I wasn't sure what that meant. Seems safe though!

Reply:
`RecentlyNotNull` is equivalent to `NotNull` but lets you incrementally update rather than breaking builds (aka no warning). You can just treat it as not null.

Reply:
👍